modules/ocf_ocfweb/manifests/dev_config.pp

```diff
@@ -7,7 +7,10 @@
   $redis_password = hiera('create::redis::password')
   validate_re($redis_password, '^[a-zA-Z0-9]*$', 'Bad Redis password')
   $ocfmail_password = hiera('ocfmail::mysql::dev_password')
-  validate_re($ocfmail_password, '^[a-zA-Z0-9]*$', 'Bad ocfmail password')
+  validate_re($ocfmail_password, '^[a-zA-Z0-9]*$', 'Bad ocfmail dev password')
+
+  $ocfstats_password = hiera('ocfstats::mysql::dev_password')
+  validate_re($ocfstats_password, '^[a-zA-Z0-9]*$', 'Bad ocfstats dev password')
```

Comment:
Unfortunately these all should follow the new format (https://github.com/ocf/puppet/commit/33b89c0159c3019d4fac0d06165618d12a089340) now.